optimum/onnxruntime/quantization.py

```diff
@@ -286,6 +287,7 @@ def quantize(
         calibration_tensors_range: Optional[Dict[str, Tuple[float, float]]] = None,
         use_external_data_format: bool = False,
         preprocessor: Optional[QuantizationPreprocessor] = None,
+        extra_options: Optional[Dict[str, Any]] = {}
```

Comment:
Can you set the default to `None` and then create an empty dict in the method please?
Dict as default value is not good because they are mutable.